package.json

```diff
@@ -65,6 +65,7 @@
     "lint": "eslint --ignore-path .gitignore --ext .js,.cjs,.mjs,.ts,.tsx .",
     "prelint": "prettier --check --ignore-path .gitignore .",
     "postlint": "stylelint '**/*.scss'",
+    "prepack": "echo \"export default '$(jq -r .version < package.json)'\" > lib/version.js",
```

Comment:
Normally I would like to use a platform independent script (can always be done later), but this will ONLY run on CI so it should be safe to do this for now.